modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -578,9 +578,10 @@ class NDB_Form_Imaging_Browser extends NDB_Form
         $depends = parent::getJSDependencies();
         if($this->page === 'viewSession') {
             $factory = NDB_Factory::singleton();
-            $baseURL = $factory->settings()->getBaseURL();
+            $baseurl = $factory->settings()->getBaseURL();
```

Comment:
Also this variable was also wrong, if the baseurl has a directory, it would fail without this fix.